server.js

```diff
@@ -24,10 +17,73 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to the Netflix data API')
+})
+
+
+// Show all data - https://netflix-dummy-data.herokuapp.com/shows
+
+app.get('/shows', (req, res) => {
+  res.json(netflixData)
 })
 
+// Show all data - 
+// https://netflix-dummy-data.herokuapp.com/shows/:id
+// example : https://netflix-dummy-data.herokuapp.com/shows/81197050
+
+app.get('/shows/:id', (req,res) => {
+  const id = req.params.id
+  let showId = netflixData.find((item) => item.show_id === +id)
+  res.json(showId)
+})
+
+// Search Title with search query -
+// We can use the query method since several movies/shows can have the same name.
+// We have to /https://netflix-dummy-data.herokuapp.com/shows/title?title=xxx
+// example: https://netflix-dummy-data.herokuapp.com/shows/title?title=cho
+
+app.get('/shows/title', (req, res) => {
+  const title  = req.query.title
+  if (title) {
+    const filteredTitle = netflixData.filter((item) => item.title === title)
+    res.json(filteredTitle)
+  } else {
+    res.json(netflixData)
+  }
+})
```

Comment:
This could be a query directly under your `shows` endpoint, like:
`/shows?title=chocolat`

PS. Would be nice with case insensitivity and letting the user search for part of a title